float64 latitude_deg 
float64 longitude_deg
float32 absolute_altidue_m
float32 relative_altitude_m